packer/endpointserver.yaml

```diff
@@ -47,7 +47,7 @@ provisioners:
       - echo Installing PHP
       - echo ------------------------------------
 
-      - sudo yum -y --enablerepo=remi,remi-php72 install php phpunit7 php-common php-mysql php-fpm php-pecl-memcache php-pecl-memcached
+      - if [ -f /etc/fedora-release ]; then sudo yum -y --enablerepo=remi install php phpunit7 php-common php-mysql php-fpm php-pecl-memcache php-pecl-memcached; else sudo yum -y --enablerepo=remi,remi-php72 install php phpunit7 php-common php-mysql php-fpm php-pecl-memcache php-pecl-memcached; fi
```

Comment:
i think that this belongs in a seperate PR?

Reply:
Why? The code in the master does not work.